CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+
+# How you can contribute
+
+Thanks for your interest in contributing to VISCtemplates! There are several ways you can contribute to this package:
+
+* **Propose an idea.** Do you have an idea for a new VISCtemplates feature? Take a look at the issue list first to see if it isn't included or suggested yet. If not, suggest your idea as an issue on GitHub.
+
+* **Report a bug.** Report a bug as an issue on GitHub so we can fix it. When filing an issue, the most important thing is to include a minimal reproducible example so that we can quickly verify the problem, and then figure out how to fix it. There are three things you need to include to make your example reproducible: required packages, data, code.
+
+    1. Packages should loaded at the top of the script.
+    2. The easiest way to include data is to use `dput()` to generate the R code to recreate it. But even better is if you can create a data.frame() with just a handful of rows and columns that still illustrates the problem.
+    3. Spend a little bit of time ensuring that your code is easy for others to read: (1) Make sure you've used spaces and your variable names are concise, but informative. (2) Use comments to indicate where your problem lies. (3) Do your best to remove everything that isn't related to the problem. The shorter your code is, the easier it is to understand.
+<br><br>
+
+* **Improve documentation.** Noticed a typo? Think a function could use a better example? Improvements to the documentation is very welcome!
+
+* **Contribute code.** If you'd like to fix a bug or implement a new feature, have a look at the issue list and leave a comment on things you want to work on. Also see the development guidelines below.
+
+
+# Development guidelines
+
+We follow the [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) development model and the guidelines from [R packages](https://r-pkgs.org/) by Hadley Wickham and Jenny Bryan.
+
+1. Create a branch in git and make your changes. 
```

Comment:
Should we say fork repo as step one or is that obvious?

Reply:
We currently don't use forks. Maybe say clone?

Reply:
Oh I thought the audience here was the general public. If it's mean to be internal feel free to disregard.

Reply:
I was thinking more internal because I don't expect anyone else being interested in contributing! But I think I'll say "clone or fork" just to be safe.

Reply:
I think I am going to leave it as-is so it is intentionally vague. If someone is familiar with git/github they should know when/how to fork and clone repos. We could always consider adding a link to an r-focused git tutorial.